### PR TITLE
Issue #34 Removing set_include_path from PersistenceByFile

### PR DESCRIPTION
**Pull request by ooglek**
As Issue #34 mentions, using `set_include_path()` has the potential to break systems.

This change removes `set_include_path()` entirely while still safely reading and writing the OAuth Tokens to a file on disk.

I added PHPUnit Tests.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"name" : "zohocrm/php-sdk",
 	"description" : "Zoho CRM API SDK for PHP",
-	"type" : "SDK",
```
Comment (ooglek) [inline composer.json:4]: Composer 1.9.1 complained that `type` must be lower-case.

```
  [Composer\Json\JsonValidationException]
  "./composer.json" does not match the expected JSON schema:
   - type : Does not match the regex pattern ^[a-z0-9-]+$
```

Composer seems to recommend leaving this unset or using `library`.
https://getcomposer.org/doc/04-schema.md#type

**Comment by ooglek** [comment]
@raja-7453 @pravesh-a Let me know if you have feedback on this Pull Request. Happy to make changes or adjust coding style to meet the project's needs.

This is affecting our production system. While we are running our fixed fork, we would like to get back to mainline as soon as possible.

Cheers!

**Comment by ooglek** [comment]
Hey @raja-7453 @pravesh-a just checking to see if you've seen the requested change. I'm happy to make any changes needed to get this fixed for everyone.

I made a quick update to remove a new property on the ByFile class, and made the temporary path for the token file in Tests to use the local directory to avoid having to create a directory.

Thanks!

**Comment by tm1000** [comment]
@ooglek FYI if you look at the history of this project they have never merged a single PR from any outside contributor. On top of that they have only ever merged 1 PR and that was from one of their own employees. You'd be best to run your fork for the foreseeable future.

**Comment by pravesh-a** [comment]
Issue fixed in SDK

**Comment by ooglek** [comment]
This was fixed, though the tests did not get pulled in.

@tm1000 It's a bummer that Zoho won't accept Pull Requests, but will happily copy the code. I mean, I don't mind terribly, but it costs nothing to acknowledge contribution, and for those contributing, it is valuable to have under your belt.

I may fork if necessary, though I'm glad the issue got fixed in under a month.

https://github.com/zoho/zcrm-php-sdk/commit/2c80249c56eb340011d5ecf569def43a35ace9fa#diff-ec1d481241ec59cfb7de0ea5f8f459ab

**Pull request closed without merging**